packages/web-frontend/src/screens/mobile/RegionScreen/index.js

```diff
@@ -293,22 +293,22 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => ({
   onChangeMapOverlay: mapOverlayId => dispatch(setMapOverlay(mapOverlayId)),
-  onUnselectMapOverlay: () => dispatch(unselectMapOverlay()),
+  onClearMeasure: () => dispatch(clearMeasure()),
   onToggleMeasureExpand: () => dispatch(toggleMeasureExpand()),
   onToggleDashboardSelectExpand: () => dispatch(toggleDashboardSelectExpand()),
   onChangeOrgUnit: organisationUnitCode => dispatch(setOrgUnit(organisationUnitCode, false)),
   onChangeDashboardGroup: name => dispatch(setDashboardGroup(name)),
 });
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  const { onChangeMapOverlay, onUnselectMapOverlay } = dispatchProps;
+  const { onChangeMapOverlay, onClearMeasure } = dispatchProps;
 
   return {
     ...stateProps,
     ...dispatchProps,
     ...ownProps,
     onChangeMapOverlay: mapOverlay =>
-      mapOverlay ? onChangeMapOverlay(mapOverlay.mapOverlayId) : onUnselectMapOverlay(),
+      mapOverlay ? onChangeMapOverlay(mapOverlay.mapOverlayId) : onClearMeasure(),
```

Comment:
Just flagging that this will clear all currently selected measures here - which may be what we want, but I'm not sure

Reply:
Yeah in [mobile.tupaia.org](mobile.tupaia.org) I couldn't find any map overlays, but correct me if I'm wrong... If so I think it should be fine to not support multi map overlay checkbox here.

Reply:
Good point - let's not worry about it then!